entrofy/mappers.py

```diff
@@ -83,11 +84,18 @@ def transform(self, column):
         nonnulls = ~column.isnull()
 
         for key in self._map:
-            df[key][nonnulls] = column[nonnulls].apply(self._map[key])
-            df[key][~nonnulls] = None
+            df['{}{}'.format(self.prefix, key)][nonnulls] = column[nonnulls].apply(self._map[key])
+            df['{}{}'.format(self.prefix, key)][~nonnulls] = None
```

Comment:
I don't see how they could. `_prepend_prefix` returns a new target dictionary with the prefixes included correctly in the keyword. These two lines add a new column to a `pandas.DataFrame` and do a mapping operation.